sdk/src/types/block/output/nft.rs

```diff
@@ -94,10 +94,10 @@ impl NftOutputBuilder {
         self
     }
 
-    /// Adds an [`UnlockCondition`] to the builder.
+    /// Adds an [`UnlockCondition`] to the builder, if one does not already exist of that type.
```

Comment:
I'm not sure I understand this comment. Maybe say it replaced it it was already there?

Reply:
It doesn't replace it, that's what `replace_unlock_condition` does.

Reply:
Oh my bad, I think I misread the doc